native/src/test/java/io/ballerina/stdlib/http/transport/websocket/server/WebSocketServerHandshakeFunctionalityTestCase.java

```diff
@@ -121,12 +121,11 @@ public void testExtensionSupport() throws IOException {
         urlConn.setRequestProperty("Sec-WebSocket-Extensions", deflateHeader);
         urlConn.setRequestProperty("x-handshake", "true");
 
-        Assert.assertEquals(urlConn.getResponseCode(), 101);
```

Comment:
Now urlConn.getResponseCode() will be error with message Unexpected 101 response from serve. MAy be we need to seperately check this.